sdk/test/utxo.test.ts

```diff
@@ -323,4 +333,38 @@ describe('UTXO Tests', () => {
             )
         );
     });
+
+    it('should not spend outputs with inscriptions', { timeout: 50000 }, async () => {
+        const paymentAddress = 'bc1peqr5a5kfufvsl66444jm9y8qq0s87ph0zv4lfkcs7h40ew02uvsqkhjav0';
+        // Use a random public key for P2SH-P2WPKH
+        const pubkey = '03b366c69e8237d9be7c4f1ac2a7abc6a79932fbf3de4e2f6c04797d7ef27abfe1';
+
+        const ordinalsClient = new OrdinalsClient('mainnet');
+
+        // cardinal = return UTXOs not containing inscriptions or runes
+        const cardinalOutputs = await ordinalsClient.getOutputsFromAddress(paymentAddress, 'cardinal');
+
+        const cardinalOutputsSet = new Set(cardinalOutputs.map((output) => output.outpoint));
+
+        const maxSpendableBalance = cardinalOutputs.reduce((acc, output) => acc + output.value, 0);
+
+        // spend 90% of max spendable amount
+        await createBitcoinPsbt(paymentAddress, paymentAddress, Math.floor(maxSpendableBalance * 0.9), pubkey);
```

Comment:
I think we should check this fails, check the balance of the address including ordinals, try to create the UTXO and check it throws an error that the account has insufficient balance. I think this test is not really doing anything useful right now since we're checking only the cardinal balance.

Reply:
This test is checking that when we create psbt none of the possible inputs contains `inscriptions` -- @sander2 wanted that to be tested.

I've added a test that checks if it throws an errors if we try to spends max balance including ordinals. I think this new test is more general and works (throws) with all types of addresses.

Reply:
Hmm actually yeah you are right that this test is still valid